source/index.html.md

```diff
@@ -690,6 +690,250 @@ surname  |  String | The ***surname***  is the last name of the document holder.
 session_id  |  uuid | The ***session_id*** uniquely identifies an enrolment. 
 
 
+# LiveProof
+
+## Retrieve Liveness Score
+
+```python
+import requests
+API_SECRET = '6e4071cb-656d-4eef-9dfd-93a7093f86b8'
```

Comment:
Can `6e4071cb-656d-4eef-9dfd-93a7093f86b8` be replaced with like `a-secret-api-key` or something similar. Otherwise, it looks like a legit one.

Reply:
Then can add something like 
`API_SECRET = '6e4071cb-656d-4eef-9dfd-93a7093f86b8'  # A non-production, example API_SECRET`

Reply:
resolved in 47d989f